renovate.json5

```diff
@@ -0,0 +1,12 @@
+{
+  extends: ['github>karlhorky/renovate-config:default.json5'],
+  // Mostly default configuration for ignorePaths
+  // https://docs.renovatebot.com/presets-default/#ignoremodulesandtests
+  ignorePaths: ['**/node_modules/**', '**/examples/**'],
```

Comment:
probably don't need `ignorePaths` at all

Reply:
7207253

Reply:
this did not remove the comments above the line, but I did this in `main` - so just take the version from there

Reply:
https://github.com/upleveled/stylelint-config-upleveled/pull/3/commits/9266ad0dc333a7486e56ce3da3b9f11c35daf390